test/controlplane/Makefile

```diff
@@ -7,3 +7,15 @@ test:
 
 update-golden:
 	go test . -test.v -update
+
+pre-pull-kind-images:
+	bash ./k8s_versions.sh --pre-pull-images
+
+update-k8s-versions:
+	# Delete all auto-generated input test files
+	find . -type d -regextype posix-extended -regex ".*/v[0-9]{1}\.[0-9]{2}" | xargs rm -r || true
+	# Update kind-config with the versions stored in k8s_versions.txt
+	$(shell bash ./k8s_versions.sh --update-kind-config)
+
+generate-input-files: pre-pull-kind-images update-k8s-versions update-golden test
```

Comment:
not sure if Makefile target is honouring the order of execution, I ran `make update-k8s-versions generate-input-files` locally to test it out, but got below error 

```
    testcase.go:252: Failed to read /home/tammach/go/src/github.com/cilium/cilium/test/controlplane/services/nodeport/v1.23/init.yaml: open /home/tammach/go/src/github.com/cilium/cilium/test/controlplane/services/nodeport/v1.23/init.yaml: no such file or directory
....
```

Reply:
oh yeah, the `update-golden test` should not be there.